### ISSUE TITLE
Attention : grep: erreur d'écriture: Relais brisé (pipe)

### ISSUE THREAD

**Issue by vincentux** [duniter/duniter_ynh#20]
voici une partie des messages au lancement d'une MAJ : 
```
Attention : dpkg : avertissement : lors de la suppression de duniter, le répertoire « /opt » n'était pas vide, donc il n'a pas été supprimé
Traitement des actions différées (« triggers ») pour mime-support (3.58) ...
+ INSTALL_DUNITER_DEBIAN_PACKAGE
++ head -1
++ grep x64
++ cut '-d"' -f4
++ grep browser_
++ curl -s https://api.github.com/repos/duniter/duniter/releases
Attention : grep: erreur d'écriture
Attention : grep: erreur d'écriture: Relais brisé (pipe)
+ url=https://github.com/duniter/duniter/releases/download/v0.30.11/duniter-v0.30.11-linux-x64.deb
++ cut -d/ -f8
++ head -1
++ grep x64
++ grep browser_
++ curl -s https://api.github.com/repos/duniter/duniter/releases
Attention : grep: erreur d'écriture
Attention : grep: erreur d'écriture: Relais brisé (pipe)
+ version=v0.30.11
+ wget -nc --quiet https://github.com/duniter/duniter/releases/download/v0.30.11/duniter-v0.30.11-linux-x64.deb -P /tmp
```

**Comment by M5oul** [comment]
I don't think it's an issue as it's working.
Thanks for the report.

**Comment by M5oul** [comment]
I close because it's not an issue.
[It's a normal behaviour with `pipe` and `head -1`](https://superuser.com/questions/554855/how-can-i-fix-a-broken-pipe-error/554896#554896).
